Cargo.toml

```diff
@@ -15,7 +15,7 @@ members = [
 
 [workspace.package]
 edition = "2021"
-version = "0.5.1"
+version = "0.6.0"
 license = "MIT OR Apache-2.0"
```

Comment:
I think properly this needs to be `(MIT OR Apache-2.0) AND OFL-1.1`, since #66. I think that the upstream change didn't make it into Bevy 0.14, unfortunately, so we can't just defer this licensing question to them.

I see that you added the proper license file in #73; thanks for that.